users/templates/registration_successful.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Registration successful</title>
+</head>
+<body>
+  <p>You successfully created a new account!</p>
+</body>
+</html>
```

Comment:
You're missing an empty line at the end of the file.

Check to see that your IDE is set up to make sure you have only one empty line at the end of each file in the project.